quadfeather/tiler.py

```diff
@@ -1471,12 +1497,13 @@ def insert_table(self, table: pa.Table):
                 child_tile.insert_table(subset)
         else:
             for sidecar, tb in self.keyed_batches(table).items():
-                for batch in tb.to_batches():
-                    try:
-                        self.overflow_buffers[sidecar].write_batch(batch)
-                    except:
-                        pass
-                        raise
+                for tb in rebatch(tb.to_batches(), 50e6):
```

Comment:
is there a reason we don't just make max_rows smaller so we can skip the inner for loop rather than rebatching and then batching again?

Reply:
IIRC rebatching here means taking something from relatively small batches up to a fixed size that's substantially larger.

Reply:
`tb.to_batches()` here is a zero-copy operation. B ut maybe I'm missing something here.

Reply:
ah gotcha, very cool